src/interactions/interaction.ts

```diff
@@ -11,8 +11,53 @@ module Plottable {
      */
     protected _componentToListenTo: Component;
 
-    public _anchor(component: Component) {
+    private _anchorCallback = (component: Component) => this._anchor(component);
+
+    private _isAnchored: Boolean;
+
+    /* tslint:disable: */
```

Comment:
Is this still needed? I tried pulling the branch and removing the `disable`, and tslint still passed.